javascript/javascript1/week1/task_2.js

```diff
@@ -0,0 +1,20 @@
+
+//dog age calculator
+
+let dogYearOfBirth = 2020;
+let dogYearFuture = 2027;
+let shouldShowResultInDogYears = true;
+
+const dogYear = dogYearFuture - dogYearOfBirth;
+
+let humanYear = dogYear / 7;
```

Comment:
you can avoid this since dogage should be calculate depend on dogYear but not humanyear.